Service/RequestService.php

```diff
@@ -548,7 +578,7 @@ public function requestHandler(array $data, array $configuration): Response
 
         $this->handleMetadataSelf($result, $metadataSelf);
 
-        $result = $this->shouldWeUnsetEmbedded($result, $this->data['headers']['accept'] ?? null, $isCollection ?? false);
+//        $result = $this->shouldWeUnsetEmbedded($result, $this->data['headers']['accept'] ?? null, $isCollection ?? false);
```

Comment:
Deze change was specifiek gedaan voor KISS (en word nu ook gebruikt voor OpenCatalogi).
Ik geloof dat deze functie was toegevoegd door @bbrands02 voor roxit zodat we alleen bij accept json+ld embedded array in de response hebben.
Door deze functie uit te commenten hebben we weer altijd een embedded array.

Rondom dit embedded verhaal moeten we misschien nog eens goed kijken naar het gebruik van extend query (doet nu niks) en ObjectEntity->toArray functie

Reply:
Moeten we deze dan wel meenemen naar de main branch?

Reply:
https://conductionworkspace.slack.com/archives/C0378UVACUT/p1675413877183459?thread_ts=1675413442.747719&cid=C0378UVACUT

Reply:
I will write a Jira issue for fixing this